.github/workflows/tests.yml

```diff
@@ -25,12 +25,14 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.x
+          dotnet-version: |
+            5.0.x
+            6.0.x
       - name: Install prerequisites and download drivers
         shell: bash
         run: ./build.sh --download-driver
       - name: Building
-        run: dotnet build ./src
+        run: dotnet build -f net5.0 ./src
       - name: Installing Browsers and dependencies
         run: dotnet run --project ./src/Playwright/Playwright.csproj -f net5.0 -- install --with-deps ${{ matrix.browser }}
```

Comment:
If we don't drop .NET 5, we should add the version to a matrix to test both versions. I would drop .NET 5.

Reply:
lets do it once .NET 5 reaches EOL or we follow-up earlier.